json_edit/plugin.spec.yaml

```
@@ -68,3 +78,25 @@ actions:
         type: '[]object'
         description: Array of objects
         required: true
+  add_key:
+    title: Add Key
```

```suggestion
    title: Add Key to Object
```

Sorry, @r7-kszczepanskagorna - this one should be add a key to JSON object. It needs an object as input too.